live-examples/css-examples/fonts/font-size-adjust.css

```diff
@@ -0,0 +1,24 @@
+@font-face {
+    font-family: 'Fira Sans';
+    src: local('FiraSans-Regular'),
+         url('/media/fonts/FiraSans-Regular.woff2') format('woff2');
+    font-weight: normal;
+    font-style: normal;
+}
+
+#output section {
+    flex-direction: column;
+}
+
+.unadjusted {
+    font-family: serif;
+    font-size: 20px;
+}
+
+.unadjusted code {
+    font-size-adjust: .45;
```

Comment:
It seems like this sets the value for the "font-size: 20px;" text, but I'm not sure why we're doing this.

I guess I'm also not sure what the meaning of the "font-size: 20px;" text is: what is it referring to? I thought initially it was referring to both "This text is unadjusted." and "This text is adjusted.", but "This text is adjusted." is only 16px.

Reply:
Yeah, this is compounding my earlier mistake. I wanted the `font-size` of everything to be the same, but the `<code>` text was enormous. This is going to go away based on your other feedback.